build/windows-script.ps1

```diff
@@ -0,0 +1,30 @@
+# Get the arch and dir
+param([String]$arch="x86_64")
+
+$NAME = "VapourSynth"
+$PY_DIR = "py-dir"
+$VS_DIR = "vs-dir"
+
+If ($arch -eq "i686") {
+        $SUFFIX = 32
+        $PYTHON_PKG = "python-3.7.5-embed-win32.zip"
+} Else {
+        $SUFFIX = 64
+        $PYTHON_PKG = "python-3.7.5-embed-amd64.zip"
+}
+
+# Download Python embeddable and VapourSynth portable
+$VS_PATH = "https://github.com/vapoursynth/vapoursynth/releases/download/R47.2"
+curl -LO "https://www.python.org/ftp/python/3.7.5/$PYTHON_PKG"
+curl -LJO "$VS_PATH/VapourSynth$SUFFIX-Portable-R47.2.7z"
+
+# Unzip Python embeddable and VapourSynth portable
+7z x "$PYTHON_PKG" -o"$PY_DIR"
+7z x "VapourSynth$SUFFIX-Portable-R47.2.7z" -o"$VS_DIR"
+
+# Move all VapourSynth files inside the Python ones
+Move-Item -Force -Path "$VS_DIR\*" -Destination "$PY_DIR"
+
+# Move the VapourSynth directory into a system directory
+Move-Item -Path "$PY_DIR" -Destination "C:\Program Files"
+Rename-Item -Path "C:\Program Files\$PY_DIR" -NewName "$NAME"
```

Comment:
I originally went with the installer route on AppVeyor because I wanted to make sure that `vapoursynth-sys`'s library folder defaults worked with the default VapourSynth installer installation. I still think this is a good idea, especially since the default installation folders are sometimes changed (for example, in R46 the 32-bit installation now happens into `C:\Program Files (x86)\VapourSynth-32`, note the `-32`). Is there any particular reason why you went with the portable installation?

Reply:
I've used this approach because the installer didn't work neither with `cmd` nor with `Powershell`. The entire run takes more than 6 hours and it was brutally interrupted as you can see [here](https://github.com/Luni-4/vapoursynth-rs/runs/273749382).

At the same time, I think the portable version gives us two advantages:

- We can install it everywhere in the system, so we are not related to specific paths and imho it's a big advantage 
- We can choose the Python embeddable version that we want to test the code

What do you think about this?

Reply:
> The entire run takes more than 6 hours and it was brutally interrupted as you can see here.

Weird. Well, let's blame Windows and move on with portable, then. :smile:

Reply:
I completely agree with you xD